node.gyp

```diff
@@ -970,7 +970,6 @@
             'src/crypto/crypto_tls.h',
             'src/crypto/crypto_clienthello.h',
             'src/crypto/crypto_context.h',
-            'src/crypto/crypto_ecdh.h',
```

Comment:
Since https://github.com/nodejs/node/pull/37048 renamed this file, shouldn't it be renamed rather than removed from the gyp definitions?

Reply:
He just forget to remove the old header file. cc file entry is right deleted.

Reply:
😕. https://github.com/nodejs/node/pull/37048/commits/7be91b505f625466e8f268b898cfdd7dc991d5df#diff-9f1d0716c75f4df39d9e88499ac205b50301c0c672b816ce54d5339be4c2238c renamed `'src/crypto/crypto_ecdh.cc',` -> `'src/crypto/crypto_ec.cc',` in `node.gyp` but forgot to rename `'src/crypto/crypto_ecdh.h',`. Nothing was deleted -- both files were only renamed.

Reply:
You were right, not sure how both passed all CI ...

Reply:
@richardlau PTAL